src/features/deeplinks/typeGuard.ts

```diff
@@ -0,0 +1,5 @@
+import { AllowedDeeplinkRoutes, DEEPLINK_TO_SCREEN_CONFIGURATION } from './types'
+
+export const isAllowedRouteTypeguard = (routeName: string): routeName is AllowedDeeplinkRoutes => {
+  return (typeof routeName === 'string' && routeName in DEEPLINK_TO_SCREEN_CONFIGURATION) ?? false
```

Comment:
Il me semble que ` (typeof routeName === 'string' && routeName in DEEPLINK_TO_SCREEN_CONFIGURATION)` est déjà un booléen. Ainsi je n'aurais pas mis `?? false` 

Qu'en penses-tu ?